packages/api/migrations/20200820002013-create-nomination.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Nominations', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      status: {
+        allowNull: false,
+        type: Sequelize.STRING
+      },
+      dateReceived: {
+        type: Sequelize.DATE
+      },
+      providerFirstName: {
+        type: Sequelize.STRING
+      },
+      providerLastName: {
+        type: Sequelize.STRING
+      },
+      providerNumber: {
+        type: Sequelize.INTEGER
+      },
+      emailAddress: {
+        type: Sequelize.STRING
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      validated: {
+        type: Sequelize.BOOLEAN
+      },
+      hospitalName: {
+        type: Sequelize.STRING
+      },
+      hospitalURL: {
+        type: Sequelize.STRING
+      },
+      hospitalAddress: {
+        type: Sequelize.STRING
+      },
+      hospitalCity: {
+        type: Sequelize.STRING
+      },
+      hospitalState: {
+        type: Sequelize.STRING
+      },
+      hospitalZipCode: {
+        type: Sequelize.INTEGER
+      },
+      representativeFirstName: {
+        type: Sequelize.STRING
+      },
+      representativeLastName: {
+        type: Sequelize.STRING
+      },
+      representativeEmailAddress: {
+        type: Sequelize.STRING
+      },
+      relationship: {
+        type: Sequelize.STRING
+      },
+      patientFirstName: {
+        type: Sequelize.STRING
+      },
+      patientLastName: {
+        type: Sequelize.STRING
+      },
+      age: {
+        type: Sequelize.INTEGER
+      },
+      admissionDate: {
+        type: Sequelize.DATE
+      },
+      dischargeDate: {
+        type: Sequelize.DATE
+      },
+      patientDiagnosis: {
+        type: Sequelize.STRING
+      },
+      amountRequested: {
+        type: Sequelize.INTEGER
+      },
+      amountGranted: {
+        type: Sequelize.INTEGER
+      },
```

Comment:
Since these are money being represented with integers, what do you think about naming them something like `amountRequestedCents` and `amountGrantedCents`, to make it clear we're storing them in cents (and they need to be converted to be displayed as dollars)?

Reply:
That sounds like a fun coding problem! Will do.